semanticdb/semanticdb/specification.md

```diff
@@ -2257,6 +2257,10 @@ In this section, we describe the Java symbol format.
         <li>
           For top-level package, its descriptor.
         </li>
+        <li>
+          For static inner classes and interfaces, concatenation of
+          of owner's simple name, dot (`.`) and definition descriptor.
```

Comment:
A simple name for `foo.Bar#` is just `Bar`, which is probably not what you wanted. Probably instead you meant "concatenation of the owner's owner symbol, owner's simple name, dot (`.`) and definition descriptor".